### PR TITLE
Vignettes small updates

### PR DESCRIPTION
**Pull request by florisvdh**
A few suggestions for `use.Rmd`.

Regarding `coerce.Rmd`: nice, no further changes made. Some figures seem a bit too low in resolution though, compared to https://rsbivand.github.io/foss4g_2022/modernizing_220822.html.

### CODE REVIEW

File: vignettes/use.Rmd [hunk 1]
```diff
@@ -83,17 +83,15 @@ Goodbye from GRASS GIS
...
 ```{r, out.width=800, echo=FALSE}
 knitr::include_graphics("rstudio_in_GRASS.png")
 ```
 
 ## Starting GRASS inside R
 
-From **spgrass6** 0.6-3, it has also been possible to start a GRASS session from a running R session using the `initGRASS()` function. This is done by setting GRASS and environment variables from the R session (https://grass.osgeo.org/grass80/manuals/variables.html). Starting GRASS from R may use a temporary location, or may use an existing GRASS `LOCATION`.
-
-### Temporary GRASS location
+From **spgrass6** 0.6-3, it has also been possible to start a GRASS session from a running R session using the `initGRASS()` function. This is done by setting GRASS and environment variables from the R session (https://grass.osgeo.org/grass80/manuals/variables.html).
```
Comment (veroandreo) [inline vignettes/use.Rmd:94]:
```suggestion
From **spgrass6** 0.6-3, it has also been possible to start a GRASS session from a running R session using the `initGRASS()` function. This is done by setting GRASS and environment variables from the R session (https://grass.osgeo.org/grass-stable/manuals/variables.html).
```

Comment (veroandreo) [reply vignettes/use.Rmd:94]: in that way it'll always redirect to the latest stable version

**Review by veroandreo — COMMENTED** [review]
Some minor comments :)

File: vignettes/use.Rmd (hunk 1 above)
Comment (rsbivand) [reply vignettes/use.Rmd:94]: Thanks!

**Pull request merged**